### DISCUSSION TITLE
Welcome to Photovoltaic_Fault_Detector Discussions!

### DISCUSSION THREAD

**Discussion by afariasfermin** [RentadroneCL/Photovoltaic_Fault_Detector#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayush7506** [comment]
Hi, I wanted to use your trained model for a similar dataset and re-train it using transfer learning but I can't find any .h5 file in the repository. How do I use this repository for predictions and re-training?

**Reply by dlsaavedra** [reply]
In section Weights of Trained Models are the files with the training weights.
I attach the link:
https://drive.google.com/drive/folders/1LSc9FkAwJrAAT8pAUWz8aax_biFAMMXS
